--- gentoo-sources-custom	2009-08-06 20:54:23.625512812 -0400
+++ gentoo-sources-i586		2009-08-06 20:54:43.375610124 -0400
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.30-gentoo-r4
-# Fri Aug  7 00:54:20 2009
+# Fri Aug  7 00:54:37 2009
 #
 # CONFIG_64BIT is not set
 CONFIG_X86_32=y
@@ -195,8 +195,8 @@
 # CONFIG_PARAVIRT_GUEST is not set
 # CONFIG_MEMTEST is not set
 # CONFIG_M386 is not set
-CONFIG_M486=y
-# CONFIG_M586 is not set
+# CONFIG_M486 is not set
+CONFIG_M586=y
 # CONFIG_M586TSC is not set
 # CONFIG_M586MMX is not set
 # CONFIG_M686 is not set
@@ -224,7 +224,7 @@
 CONFIG_X86_L1_CACHE_BYTES=64
 CONFIG_X86_INTERNODE_CACHE_BYTES=64
 CONFIG_X86_CMPXCHG=y
-CONFIG_X86_L1_CACHE_SHIFT=4
+CONFIG_X86_L1_CACHE_SHIFT=5
 CONFIG_X86_XADD=y
 # CONFIG_X86_PPRO_FENCE is not set
 CONFIG_X86_F00F_BUG=y
