--- zen-sources-custom	2009-08-06 21:05:07.979091252 -0400
+++ zen-sources-i586	2009-08-06 21:06:31.967256516 -0400
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.30-zen1 "Summertime Tropic Eclipse"
-# Fri Aug  7 01:04:51 2009
+# Fri Aug  7 01:06:17 2009
 #
 # CONFIG_64BIT is not set
 CONFIG_X86_32=y
@@ -128,7 +128,7 @@
 #
 # Custom Build Flags
 #
-CONFIG_CUSTOM_CFLAGS="-march=i486 -O2 -pipe"
+CONFIG_CUSTOM_CFLAGS="-march=i586 -Os -pipe"
 CONFIG_CUSTOM_MAKEFLAGS=""
 CONFIG_SYSCTL=y
 CONFIG_ANON_INODES=y
@@ -150,10 +150,12 @@
 CONFIG_EVENTFD=y
 CONFIG_SHMEM=y
 CONFIG_AIO=y
+CONFIG_HAVE_PERF_COUNTERS=y
 
 #
 # Performance Counters
 #
+# CONFIG_PERF_COUNTERS is not set
 CONFIG_VM_EVENT_COUNTERS=y
 CONFIG_PCI_QUIRKS=y
 CONFIG_COMPAT_BRK=y
@@ -223,8 +225,8 @@
 # CONFIG_PARAVIRT_GUEST is not set
 # CONFIG_MEMTEST is not set
 # CONFIG_M386 is not set
-CONFIG_M486=y
-# CONFIG_M586 is not set
+# CONFIG_M486 is not set
+CONFIG_M586=y
 # CONFIG_M586TSC is not set
 # CONFIG_M586MMX is not set
 # CONFIG_M686 is not set
@@ -253,7 +255,7 @@
 CONFIG_X86_L1_CACHE_BYTES=64
 CONFIG_X86_INTERNODE_CACHE_BYTES=64
 CONFIG_X86_CMPXCHG=y
-CONFIG_X86_L1_CACHE_SHIFT=4
+CONFIG_X86_L1_CACHE_SHIFT=5
 CONFIG_X86_XADD=y
 # CONFIG_X86_PPRO_FENCE is not set
 CONFIG_X86_F00F_BUG=y
